Cargo.lock

```diff
@@ -3270,28 +3411,23 @@ dependencies = [
 ]
 
 [[package]]
-name = "tinyvec"
-version = "1.8.0"
+name = "tinystr"
+version = "0.7.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "445e881f4f6d382d5f27c034e25eb92edd7c784ceab92a0937db7f2e9471b938"
+checksum = "9117f5d4db391c1cf6927e7bea3db74b9a1c1add8f7eda9ffd5364f40f57b82f"
 dependencies = [
- "tinyvec_macros",
+ "displaydoc",
+ "zerovec",
 ]
 
-[[package]]
-name = "tinyvec_macros"
-version = "0.1.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1f3ccbac311fea05f86f61904b462b55fb3df8837a366dfc601a0161d0532f20"
-
 [[package]]
 name = "tokio"
-version = "1.40.0"
+version = "1.42.0"
```

Comment:
It looks like all packages were updated? (since I don't think tokio is a dependency of `url` or `hashbrown`?).

Btw i think it's fine to update, it's just not mentioned.

Reply:
Ups, no that's not fine. Fill fix.

Reply:
Or maybe is. I hardcoded versions of url and hashbrown, and ran cargo update, so that this will be upgraded automatically once new versions come out.

Reply:
Can't you do:
```
cargo update -p url
cargo update -p hashbrown
```
if you want to update just these two? (and their dependencies).

Reply:
Yes, saw that and making changes